core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooQueueLock.java

```diff
@@ -32,7 +33,7 @@
 
 public class ZooQueueLock implements QueueLock {
 
-  private static final String PREFIX = "lock-";
+  private static final String PREFIX = "zlock#" + UUID.randomUUID() + "#";
```

Comment:
If it is required that this PREFIX here and the ZLOCK_PREFIX in ZooLock match, then maybe the definition should be placed in Constants and shared.  (Not saying that the should be the same, that's a different discussion)

Reply:
I think the parts that have to match is the "zlock#" and then also end in a "#". The first part could definitely be placed in Constants. 

I guess the issue is the `CreateTable` command uses `ZooQueueLock` for creation and the `FateCommand` uses `ZooLock` for validation.

Reply:
Constants.java has slowly been in the process of being phased out, as constants move closer to the code that is responsible for them, under the premise that fewer global constants increase modularity and separation of responsibilities.

If the schema is defined by ZooLock and ZooQueueLock is trying to match it, then the constant should live in ZooLock.